_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -85,6 +85,9 @@ Note that this is only support to run Foreman and Foreman Proxy themselves on EL
 #### Foreman - Compute resources - oVirt
 * Foreman error on retrieving networks/vnics from oVirt if description contains non ascii characters \- [#32908](https://projects.theforeman.org/issues/32908)
 
+#### Foreman - DHCP
+* Report nicely when subnet has invalid address \- [#33929](https://projects.theforeman.org/issues/33929)
```

Comment:
I know this is the title, so unsure if you want to change or not, but 
```suggestion
* Provide a meaningful report when a subnet has invalid address \- [#33929](https://projects.theforeman.org/issues/33929)
```
I think this better explains. It is not a blocker in my view.